src/features/identityCheck/pages/helpers/computeIdentificationMethod.native.test.ts

```diff
@@ -0,0 +1,44 @@
+import { IdentityCheckMethod } from 'api/gen'
+import { computeIdentificationMethod } from 'features/identityCheck/pages/helpers/computeIdentificationMethod'
+import { SubscriptionRootStackParamList } from 'features/navigation/RootNavigator/types'
+
+const educonnectFlow: (keyof SubscriptionRootStackParamList)[] = [
+  'IdentityCheckEduConnect',
+  'IdentityCheckEduConnectForm',
+  'IdentityCheckValidation',
+]
+
+const ubbleFlow: (keyof SubscriptionRootStackParamList)[] = ['SelectIDOrigin']
+const ubbleForkFlow: (keyof SubscriptionRootStackParamList)[] = ['IdentificationForkUbble']
+const educonnectForkFlow: (keyof SubscriptionRootStackParamList)[] = [
+  'IdentificationForkEduconnect',
+]
+
+const educonnectOnlyMethod = [IdentityCheckMethod.educonnect]
+const ubbleOnlyMethod = [IdentityCheckMethod.ubble]
+const bothMethodsUbbleFirst = [IdentityCheckMethod.ubble, IdentityCheckMethod.educonnect]
+const bothMethodsEduconnectfirst = [IdentityCheckMethod.educonnect, IdentityCheckMethod.ubble]
+
+describe('computeIdentificationMethod', () => {
+  it('should return a flow', () => {
+    // Fork with A/B testing
+    expect(computeIdentificationMethod(bothMethodsUbbleFirst, IdentityCheckMethod.ubble)).toEqual(
+      ubbleForkFlow
+    )
+    expect(
+      computeIdentificationMethod(bothMethodsUbbleFirst, IdentityCheckMethod.educonnect)
+    ).toEqual(educonnectForkFlow)
+    expect(
+      computeIdentificationMethod(bothMethodsEduconnectfirst, IdentityCheckMethod.ubble)
+    ).toEqual(ubbleForkFlow)
+    expect(
+      computeIdentificationMethod(bothMethodsEduconnectfirst, IdentityCheckMethod.educonnect)
+    ).toEqual(educonnectForkFlow)
+    // Only Ubble allowed
+    expect(computeIdentificationMethod(ubbleOnlyMethod)).toEqual(ubbleFlow)
+    // Only Educonnect allowed
+    expect(computeIdentificationMethod(educonnectOnlyMethod)).toEqual(educonnectFlow)
+    // Default case
+    expect(computeIdentificationMethod()).toEqual(ubbleFlow)
+  })
```

Comment:
J'aurais bien vu un it par expect, genre
 - it('should return a/b testing ubble fork when ...
- it('should return educonnect flow when ...
etc